ports/espressif/mpconfigport.mk

```diff
@@ -45,6 +45,7 @@ CIRCUITPY_SYNTHIO_MAX_CHANNELS ?= 12
 CIRCUITPY_TOUCHIO_USE_NATIVE ?= 1
 CIRCUITPY_WATCHDOG ?= 1
 CIRCUITPY_WIFI ?= 1
+CIRCUITPY_UVC ?= 1
```

Comment:
```suggestion
CIRCUITPY_USB_UVC ?= 1
```

Reply:
Do you want the whole module named `usb_uvc`?

Reply:
That wasn't actually my intent with this comment. All of the other locations were already `CIRCUITPY_USB_UVC` I was just making this consistent. I do think `usb_video` is a better name though.